js/foam/browser/u2/BrowserView.js

```diff
@@ -19,431 +19,357 @@ CLASS({
-          code: function() {
-            this.searchMode = false;
-            this.data.search = '';
-          }
-        },
-        {
-          name: 'createButton',
-          ligature: 'add',
-          isAvailable: function() { return this.data.showAdd; },
-          code: function() {
-            this.data.createFunction.call(this);
-          }
-        },
-      ],
+  listeners: [
+    {
+      name: 'onMenuTouch',
+      documentation: 'Called when a menu item is selected.',
+      code: function() {
+        this.menuOpen = false;
+      }
+    },
+    {
+      name: 'onViewDestroyed',
+      documentation: 'Called when the child view (detail view) is closed.',
+      code: function(sender, topic, view) {
+        if (view.data && this.selection && view.data.id === this.selection.id) {
+          this.selection = null;
+        }
+      }
+    },
+  ],
 
-      listeners: [
-        {
-          name: 'onMenuTouch',
-          documentation: 'Called when a menu item is selected.',
-          code: function() {
-            this.menuOpen = false;
-          }
-        },
-        {
-          name: 'onViewDestroyed',
-          documentation: 'Called when the child view (detail view) is closed.',
-          code: function(sender, topic, view) {
-            if (view.data && this.selection && view.data.id === this.selection.id) {
-              this.selection = null;
-            }
-          }
-        },
-      ],
+  templates: [
+    function CSS() {/*
+      $ {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        position: relative;
+      }
+      $-header {
+        align-items: center;
+        color: #fff;
+        display: flex;
+        flex-grow: 0;
+        flex-shrink: 0;
+        height: 56px;
+        padding: 0;
+      }
+      $-header-title {
+        flex-grow: 1;
+        font-size: 20px;
+        font-weight: 500;
+        letter-spacing: .02em;
+        margin-left: 12px;
+      }
 
-      templates: [
-        function CSS() {/*
-          $ {
-            display: flex;
-            flex-direction: column;
-            height: 100%;
-            position: relative;
-          }
-          $-header {
-            align-items: center;
-            color: #fff;
-            display: flex;
-            flex-grow: 0;
-            flex-shrink: 0;
-            height: 56px;
-            padding: 0;
-          }
-          $-header-title {
-            flex-grow: 1;
-            font-size: 20px;
-            font-weight: 500;
-            letter-spacing: .02em;
-            margin-left: 12px;
-          }
+      $-search-header .foam-u2-md-Input {
+        flex-grow: 1;
+      }
+      $-search-header input {
+        color: #fff;
+      }
 
-          $-search-header .foam-u2-md-Input {
-            flex-grow: 1;
-          }
-          $-search-header input {
-            color: #fff;
+      <% var prefixes = ['::-webkit-input', ':-moz', '::-moz', ':-ms-input'];
```

Comment:
We definitely don't support this right now.

I talked to Mark, but this is a messy change to the CSS parser. I've filed #493 to track it, we'll get to it eventually.